rest-lib/src/main/scala/com/gu/mediaservice/lib/auth/Permissions.scala

```diff
@@ -14,7 +14,9 @@ object Permissions {
   type PrincipalFilter = Principal => Boolean
 
   case object EditMetadata extends SimplePermission
+  case object UploaderEditMetadata extends SimplePermission
```

Comment:
Let's start putting comments on these so it's easier to understand what their intention is.